src/logic/calculate.js

```diff
@@ -0,0 +1,28 @@
+import calc from './operate';
+
+const method = (() => {
+  const calculate = (calcObj, buttonName) => {
+    const obj = calcObj;
```

Comment:
there is no point creating another obj var, you should destructure calcObj get the content in it